rest-api-spec/src/main/resources/rest-api-spec/test/index/110_constant_keyword.yml

```diff
@@ -68,3 +66,267 @@
   - do:
       indices.delete:
         index: test
+
+---
+"Queries":
+  - skip:
+      version: " - 2.99.99"
```

Comment:
> @reta please advise, should this be set to ` - 2.16.99`, or ` - 2.99.99` is ok?

@kkewwei technically, it should be ` - 2.16.99` but the test might fail because the change is not in `2.x` yet, so 2.99.99 is OK